cmd/translatesfx/README.md

```diff
@@ -1,5 +1,11 @@
 # SignalFx Smart Agent Configuration Translation Tool
 
+> # :warning: End of Support (EoS) Notice
+> **The SignalFx Smart Agent has reached End of Support.**
+>
+> The [Splunk Distribution of OpenTelemetry Collector](https://docs.splunk.com/Observability/gdi/opentelemetry/opentelemetry.html) is the successor. Smart Agent monitors are available and supported through the [Smart Agent receiver](https://docs.splunk.com/Observability/gdi/opentelemetry/components/smartagent-receiver.html) in the Splunk Distribution of OpenTelemetry Collector.
+>
+
```

Comment:
May be helpful to not require running the tool or seeing the updated product doc when it lands for good measure:

```suggestion

## Notice: `translatesfx` is deprecated and will be removed from the distribution of the collector in an upcoming release.
```

Reply:
Oh, so we'd be removing the tool entirely in 0.90?

Reply:
No, in an "upcoming release".